### PR TITLE
TASK: Fusion use glob includes.

### PR DESCRIPTION
**Pull request by mhsdesign**
this makes it easier for us to maintain those lists. As when adding or removing prototypes this can be forgotten.

### CODE REVIEW

**Comment by mhsdesign** [comment]
reparis partially: https://github.com/neos/neos-development-collection/pull/3649

File: Neos.Neos/Resources/Private/Fusion/DefaultFusion.fusion [hunk 1]
```diff
@@ -1,29 +1,4 @@
...
-include: Prototypes/BreadcrumbMenu.fusion
-include: Prototypes/DimensionsMenu.fusion
-include: Prototypes/Menu.fusion
-include: Prototypes/MenuItems.fusion
-include: Prototypes/BreadcrumbMenuItems.fusion
-include: Prototypes/DimensionsMenuItems.fusion
-include: Prototypes/Plugin.fusion
-include: Prototypes/PluginView.fusion
-include: Prototypes/ConvertUris.fusion
-include: Prototypes/ConvertNodeUris.fusion
```
Comment (mhsdesign) [inline Neos.Neos/Resources/Private/Fusion/DefaultFusion.fusion:18]: this is the bösewicht

**Review by jonnitto — APPROVED** [review]

File: Neos.Fusion/Resources/Private/Fusion/Root.fusion [hunk 2]
```diff
@@ -145,7 +143,7 @@ prototype(Neos.Fusion:UriBuilder) {
 #   class = 'some-class'
 # }
 #
-prototype(Neos.Fusion:Augmenter) < prototype(Neos.Fusion:Component) {
```
Comment (kdambekalns) [inline Neos.Fusion/Resources/Private/Fusion/Root.fusion:148]: Huh, this is unrelated, strictly speaking. Also, I (personally) dislike prototypes that "have no type", I always wonder what they actually are…

Why was this change done?

Comment (mhsdesign) [reply Neos.Fusion/Resources/Private/Fusion/Root.fusion:148]: youre right - this change was just because i saw this laying around - will revert ...

 it is just totally useless to have inheritance + an own `@class`
the `@class` will overrule the inheritance and we dont need inheritance in this case as `Neos.Fusion:Augmenter` has nothing to do with Fusion:Component

Comment (mhsdesign) [reply Neos.Fusion/Resources/Private/Fusion/Root.fusion:148]: `prototype(Neos.Fusion:Component)` would also "have no type" ^^

Comment (kdambekalns) [reply Neos.Fusion/Resources/Private/Fusion/Root.fusion:148]: Oh, yes. I should have ntoiced the `@class`… fine with me then.

**Review by kdambekalns — APPROVED** [review]
